java/junit4/pom.xml

```diff
@@ -2,178 +2,213 @@
+    </licenses>
 
-  <organization>
-    <name>Sauce Labs</name>
-    <url>https://www.saucelabs.com/</url>
-  </organization>
-  <developers>
-    <developer>
-      <name>Titus Fortner</name>
-      <email>titus@saucelabs.com</email>
-      <organization>Sauce Labs</organization>
-      <organizationUrl>https://www.saucelabs.com/</organizationUrl>
-    </developer>
-    <developer>
-      <name>Nikolay Advolodkin</name>
-      <email>nikolay.advolodkin@saucelabs.com</email>
-      <organization>Sauce Labs</organization>
-      <organizationUrl>https://www.saucelabs.com/</organizationUrl>
-    </developer>
-  </developers>
+    <organization>
+        <name>Sauce Labs</name>
+        <url>https://www.saucelabs.com/</url>
+    </organization>
+    <developers>
+        <developer>
+            <name>Titus Fortner</name>
+            <email>titus@saucelabs.com</email>
+            <organization>Sauce Labs</organization>
+            <organizationUrl>https://www.saucelabs.com/</organizationUrl>
+        </developer>
+        <developer>
+            <name>Nikolay Advolodkin</name>
+            <email>nikolay.advolodkin@saucelabs.com</email>
+            <organization>Sauce Labs</organization>
+            <organizationUrl>https://www.saucelabs.com/</organizationUrl>
+        </developer>
+    </developers>
 
-  <scm>
-    <connection>scm:git:git://github.com/saucelabs/sauce_bindings.git</connection>
-    <developerConnection>scm:git:git@github.com:saucelabs/sauce_bindings.git</developerConnection>
-    <url>git@github.com:saucelabs/sauce_bindings.git</url>
-    <tag>HEAD</tag>
-  </scm>
+    <scm>
+        <connection>scm:git:git://github.com/saucelabs/sauce_bindings.git</connection>
+        <developerConnection>scm:git:git@github.com:saucelabs/sauce_bindings.git</developerConnection>
+        <url>git@github.com:saucelabs/sauce_bindings.git</url>
+        <tag>HEAD</tag>
+    </scm>
 
-  <properties>
-    <maven.compiler.source>11</maven.compiler.source>
-    <maven.compiler.target>11</maven.compiler.target>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-  </properties>
+    <properties>
+        <saucebindings-junit4.version>2.0.0-beta.1-SNAPSHOT</saucebindings-junit4.version>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
 
-  <dependencies>
-    <dependency>
-      <groupId>com.saucelabs</groupId>
-      <artifactId>sauce_bindings</artifactId>
-      <version>1.5.0</version>
-    </dependency>
-    <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <version>4.13.2</version>
-    </dependency>
-  </dependencies>
+    <dependencies>
+        <dependency>
+            <groupId>com.saucelabs</groupId>
+            <artifactId>sauce_bindings</artifactId>
+            <version>2.0.0-beta.1-SNAPSHOT</version>
```

Comment:
Add note to README that this needs to be kept up to date with current version at HEAD for purposes of testing in GitHub